client/views/calendar/calendar.html

```diff
@@ -2,17 +2,11 @@
 	{{> calendarNav date=startDate}}
 	<div class="container calendar-container">
 		<div class="page-component">
-			{{> calendarAddEvent}}
-		</div>
-		<div class="page-component">
```

Comment:
Users asked for link to create events here in the calendar. shall we instead link course creation here?

Reply:
My opinion is that we either include link to create course (instead of event) or let it separate for now since this is already the 2nd menu option (Create). I don't think that this is such a big inconvenience to have a separate "View" and "Create" pages.